server/routes/api.js

```diff
@@ -4,14 +4,15 @@ import { middlewares } from 'auth0-extension-express-tools';
 import config from '../lib/config';
 import connections from './connections';
 import applications from './applications';
+import groups from './groups';
 
 export default (storage) => {
   const api = Router();
 
   // Allow end users to authenticate.
   api.use(middlewares.authenticateUsers.optional({
     domain: config('AUTH0_DOMAIN'),
-    audience: config('EXTENSION_CLIENT_ID'),
+    audience: config('API_AUDIENCE') || 'urn:sso-dashboard-api',
     credentialsRequired: false,
     onLoginSuccess: (req, res, next) => {
       const currentRequest = req;
```

Comment:
In here we need to check that only clients can have the manage:applications scope. Example: https://github.com/auth0/auth0-authorization-extension/blob/9307398a99fbbe7ad88c914d10ce3ddc3e42f0b5/server/plugins/auth.js#L84